utilities/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - python
     - iris
     - lxml
+    - owslib
+    - requests
```

Comment:
@rsignell-usgs yep, when the version is not changing we have to remember to increment it.  (But `utilities` version will change soon :smile: https://github.com/pyoceans/utilities/pull/34)

@pelson Just an idea.  I build a lot of stuff using OpenSUSE build service and the system compare the spec file changes, using a version control system, and increment the build number automatically.  Maybe that could be added to obvious-ci?  Or maybe it is a lot of work for such a simple thing...